contracts/protocol/PositionManager.sol

```diff
@@ -288,28 +288,26 @@ contract PositionManager is
             );
             hasLiquidity = liquidityBitmap.hasLiquidity(_pip);
         }
+        uint128 remainingSize = _size - uint128(sizeOut);
```

Comment:
remaining size of limit order after openMarketOrder (if this order filled current pip)